<launch>
	<!-- vim: set ft=xml noet : -->

	<node pkg="land" type="landTargetPosition" name="landTargetPosition" output="screen">
    <node pkg="imav" type="gameplan_land" name="gameplan_land" output="screen">
		<param name="height_land" type="double" value="1.0" />
	</node>
</launch>
